build/ci/community/check_all_checks.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/bash -xe
 set +o pipefail
 
-export passed_k8s_checks=true
-export passed_openshift_checks=true
+# CSI-3172 - run rad hat bot checks
```

Comment:
```suggestion
# CSI-3172 - run Red Hat bot checks
```

Reply:
`rad hat` would have been a much cooler name for them